org.adempiere.base/src/org/compiere/model/MArchive.java

```diff
@@ -59,28 +61,16 @@ public class MArchive extends X_AD_Archive {
 	 * @return archives
 	 */
 	public static MArchive[] get(Properties ctx, String whereClause) {
-		ArrayList<MArchive> list = new ArrayList<MArchive>();
-		StringBuilder sql = new StringBuilder("SELECT * FROM AD_Archive WHERE AD_Client_ID=?");
-		if (whereClause != null && whereClause.length() > 0)
+		List<MArchive> list = new ArrayList<MArchive>();

```

Comment:
this should be simplified to List<MArchive> list = new Query ... below.

Reply:
Done.